packages/p2p/src/components/my-profile/my-profile.scss

```diff
@@ -37,5 +37,9 @@
     &__navigation {
         display: flex;
         justify-content: space-between;
+
+        &:last-child {
+            margin-bottom: 1.6rem;
+        }
```

Comment:
The `MyProfileSeparatorContainer.Line` component is not shown for the last tab, thus there is no margin offset for the last tab since `MyProfileSeparatorContainer.Line` (which contains the top/bottom margin offset between each tabs) is not rendered. This component has a margin top and bottom of `1.6rem`, adding bottom margin offset manually for `my-profile__navigation` to substitute the missing component offset styling